infra-templates/ipsec/16/README.md

```diff
@@ -6,11 +6,10 @@ Rancher networking plugin using IPsec.
 
 Traffic to and from hosts require UDP ports `500` and `4500` to be open.
 
-### Changelog - 0.2.2
+### Changelog - 0.2.3
 
-#### Router and CNI Driver [rancher/net:v0.13.7]
-* Introduced connectivity check for ipsec.
-* Fixed an issue where scheduler IPs could be picked up as the IP for the ipsec service for ports `500` and `4500`, which would cause cross host communication to no longer work for that host
+#### Router, CNI Driver, Connectivity Check [rancher/net:v0.13.8]
+* Added support for regions
```

Comment:
can you add the change log for the logs update? https://github.com/rancher/rancher/issues/7571